solidity/contracts/hooks/ArbL2ToL1Hook.sol

```diff
@@ -35,13 +36,14 @@ import {ArbSys} from "@arbitrum/nitro-contracts/src/precompiles/ArbSys.sol";
  */
 contract ArbL2ToL1Hook is AbstractMessageIdAuthHook {
     using StandardHookMetadata for bytes;
+    using Message for bytes;
 
     // ============ Constants ============
 
     // precompile contract on L2 for sending messages to L1
     ArbSys public immutable arbSys;
-    // Immutable quote amount
-    uint256 public immutable GAS_QUOTE;
+    // child hook to call first
+    AbstractPostDispatchHook public immutable childHook;
```

Comment:
using the `Abstract` as the interface is a bit weird
can we just use the `IPostDispatchHook`?